### DISCUSSION TITLE
Understanding artillery fargate executions.

### DISCUSSION THREAD

**Discussion by Thilaknath** [artilleryio/artillery#2223] [Artillery]
Hello Team, 

We have a couple of teams in our organizations using artillery to execute load test scenarios. In this case would it be possible for the teams to execute the tests under one AWS account or do we need to create accounts for specific teams? The teams want to make sure their performance runs don't hinder or consume resources from another teams run being in a single AWS account. Can you please help clarify this?

**Comment by manuelcarriernunes** [comment]
@Thilaknath, I also created a discussion for this 😆

**Reply by Thilaknath** [reply]
Related question here: https://github.com/artilleryio/artillery/discussions/2224

**Reply by hassy** [reply]
Hi @Thilaknath 👋 Artillery creates completely separate sets of AWS resources for different load test runs. There's no sharing between different load tests so they won't affect each other. As long as the total quota for things like concurrent Fargate tasks is high enough to accommodate everyone, you should be fine.

**Reply by Thilaknath** [reply]
Thanks @hassy . Does passing the `--cluster` option allow us to target a Amazon EKS which we will have to provisioned ourselves prior to running the tests?

**Reply by hassy** [reply]
@Thilaknath No we don't support EKS yet, only ECS/Fargate.
